README.md

```diff
@@ -90,6 +97,14 @@ You do not have sufficient access to the device. You probably want to add the a
 
 <!-- markdownlint-enable MD013 -->
 
+## Interference from CUPS
+
+Sometimes, the CUPS printing system will attempt to communicate with the label
+maker, causing interference with Labelle.  If you don't otherwise have a USB
+printer, you can disable CUPS attempting this with:
+
+sudo touch /etc/udev/rules.d/70-printers.rules
```

Comment:
Could you please add a fenced block around this command?

Reply:
Yes, will push shortly.